lights.ino

```diff
@@ -0,0 +1,14 @@
+void fadeToALevel(int fadeLevel){
+	analogWrite(pin,fadeLevel);
```

Comment:
You didn't declare fadeLevel in main.ino but then you don't call that function anyway so it is redundant code.